gradle/libs.versions.toml

```diff
@@ -12,10 +12,10 @@ moshi = "1.15.0"
 minSdk = "25"
 compileSdk = "33"
 
-# Maps to this commit: https://android.googlesource.com/platform/prebuilts/studio/layoutlib/+/5128371
+# Maps to this tag: https://android.googlesource.com/platform/prebuilts/studio/layoutlib/+/refs/tags/studio-2022.2.1
 # which runs on JDK 11.
 layoutlib = "2022.2.1-5128371-2"
-layoutlibPrebuiltSha = "5128371"
+layoutlibPrebuiltSha = "2022.2.1"
```

Comment:
@jrodbx I need help with these lines, not sure in what dependency order or name this should be.

See also `String tag = "refs/tags/studio-${libs.versions.layoutlibPrebuiltSha.get()}"` line in build.gradle.

Reply:
https://github.com/cashapp/paparazzi/pull/884/commits/f7cc4d887e51277279e3fc7bfea9981520977857